_gitbook/conventions/coding_style.md

```diff
@@ -0,0 +1,140 @@
+# Coding Style
+
+This style is used in the standard library. You can use it in your own project to make it familiar to other developers.
+
+## Naming
+
+__Type names__ are camelcased. For example:
```

Comment:
Grammatically it should be camel-case or CamelCase. However it can be a method name, but better to add single backticks around it like `camelcase`.